trailbot_interfaces/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.8)
```

Comment:
Did this package make it in here by mistake? I think this should just be the voice assistant package.

Reply:
voice_assistant_pkg depends on trailbot interfaces so voice_assistant_pkg branch branches out of trailbot interfaces branch. If you just pull voice_assistant_pkg branch it won't build.